### PR TITLE
update dependencies, farao-core v3.5.0

### PR DESCRIPTION
**Pull request by baptsegu**
farao-core v3.4.3
farao-core-rte v2.4.4
powysbl-core v4.4.0
powsybl-rte-core v3.4.0
powsybl-olf v0.13.0

Major release of farao-core.

+ addition of a technical dependency necessary to run TU with spring on some of our computers.

### CODE REVIEW

File: farao-parent/farao-parent-ws/pom.xml [hunk 1]
```diff
@@ -54,13 +55,22 @@
...
                 <scope>import</scope>
             </dependency>
+
+            <!-- Test dependencies -->
+            <dependency>
+                <groupId>org.junit.platform</groupId>
+                <artifactId>junit-platform-launcher</artifactId>
+                <version>${junit.launcher.version}</version>
+                <scope>test</scope>
+            </dependency>
```
Comment (TheMaskedTurtle) [inline farao-parent/farao-parent-ws/pom.xml:73]: We want to reduce at most the number of dependencies defined here and we discuss everytime we want to add a new one. It looks like this is used to handle a very specific problem for some developers that can be can fixed other ways so this won't be added to farao-parent.

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

**Comment by pet-mit** [comment]
can we already update to farao-core 3.5.0 / farao-core-rte 2.5.0 please ? :D

File: farao-parent/farao-parent-ws/pom.xml (hunk 1 above)
Comment (baptsegu) [reply farao-parent/farao-parent-ws/pom.xml]: removed. To add in child-repo instead, when needed

**Comment by baptsegu** [comment]
updated to farao-core 3.5.0 / farao-core-rte 2.5.0

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**